infrastructure/src/main/java/org/corfudb/infrastructure/FailureHandlerDispatcher.java

```diff
@@ -32,14 +32,18 @@
      * @param corfuRuntime   Connected runtime
      * @return True if the cluster was recovered, False otherwise
      */
-    public boolean recoverCluster(Layout recoveryLayout, CorfuRuntime corfuRuntime) {
+    public boolean recoverCluster(ServerContext serverContext, Layout recoveryLayout,
+                                  CorfuRuntime corfuRuntime) {
 
         try {
             // Seals and increments the epoch.
             recoveryLayout.setRuntime(corfuRuntime);
             sealEpoch(recoveryLayout);
             log.info("After seal: {}", recoveryLayout.asJSONString());
 
+            // Mark cluster as ready after sealing.
```

Comment:
Sorry, this comment should be "Mark node as ready after sealing"
The status is just communicated via the shared context between the different components on the same node.